cliquet/tests/resource/test_events.py

```diff
@@ -0,0 +1,34 @@
+import webtest
+from pyramid.config import Configurator
+
+from cliquet.events import CollectionChanged
+from cliquet.tests.testapp import main as testapp
+from cliquet.tests.support import unittest, BaseWebTest, get_request_class
+
+
+class EventsTest(BaseWebTest, unittest.TestCase):
+
+    def get_test_app(self, settings=None):
+        settings = self.get_app_settings(settings)
+        self.config = Configurator(settings=settings)
+        app = testapp(config=self.config)
+        app = webtest.TestApp(app)
+        app.RequestClass = get_request_class(self.api_prefix)
+        return app
+
+    def test_events_triggered(self):
+        # let's subcribe to the CollectionChanged event
+        events = []
+
+        def listener(event):
+            events.append(event)
+
+        self.config.add_subscriber(listener, CollectionChanged)
```

Comment:
Using a mock it would like (totally untested):
```
listener = mock.Mock()
self.config.add_subscriber(listener, CollectionChanged)
...
self.app.get(..)
self.assertFalse(listener.called)
```
or
```
self.app.post(..)
self.assertTrue(listener.called)
```

and if you want to inspect events:
```
args, kwargs = listener.callargs_list[-1]
self.assertEqual(args[0], 'create')
```